app/routes.py

```diff
@@ -1 +1,149 @@
-from flask import Blueprint
\ No newline at end of file
+from app import db
+from app.models.task import Task
+from app.models.goal import Goal
+from flask import Blueprint, jsonify, abort, make_response, request
+from sqlalchemy import desc, asc
+from datetime import date
+import requests
+# import slack
+import os
+from pathlib import Path
+from dotenv import load_dotenv
+
+# env_path = Path('.') / '.env'
+# load_dotenv(dotenv_path=env_path)
+# client = slack.WebClient(token=os.environ['SLACK_TOKEN'])
```

Comment:
Generally, we should remove comments and code that aren't intended to be added to production.